cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -208,3 +224,24 @@ def test_reports_disable_enable_schedule_from_summary(appliance, schedule):
     navigate_to(schedule, "Details")
     view.configuration.item_select("Enable this Schedule")
     assert schedule.enabled
+
+
+def test_reports_schedules_user(appliance, request, user, schedule_data):
+    """
+    This test checks if a user is visible under the Emails options of schedule form
+    while creating a schedule
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/10h
+        setup:
+            1. Create a user with an email belonging to the same group as logged in user.
+        testSteps:
+            1. Create a schedule and check if the newly created user is available
```

Comment:
I don't see how this is actually validated here.

The exists method for the Schedule is just navigating to details page and using is_displayed, which from what I see is not validating the user or any of its components, its only validating the schedule name.

In a test like this, the validation needs to be explicit, not reliant on the framework methods to provide specific widget validation. The framework components could change, impacting the test coverage.

Reply:
> In a test like this, the validation needs to be explicit, not reliant on the framework methods to provide specific widget validation. The framework components could change, impacting the test coverage.

Hmm, sounds right! I'll add some more validation.